model/test/SensorMountTests.py

```diff
@@ -0,0 +1,196 @@
+
+        mount_location = (0, 0, 0)
+        rotated_location = (-1, 0, 0)
+        mount_under_test.add_new_sensor(attached_sensor)
+        mount_under_test.move_to_position(mount_location, 180)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[0],\
+            rotated_location[0], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[1],\
+            rotated_location[1], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[2],\
+            rotated_location[2], 6)
+
+    def test270DegreeRotation(self):
+        """
+        Confirm that a SensorMount with a single sensor properly rotates its
+        Sensor object's location when it is rotated 270 degrees without a
+        translation.
+        """
+        mount_under_test = SensorMount()
+
+        sensor_location = (1, 0, 0)
+        attached_sensor = LightSensor(sensor_location, (), ())
+
+        mount_location = (0, 0, 0)
+        rotated_location = (0, 1, 0)
+        mount_under_test.add_new_sensor(attached_sensor)
+        mount_under_test.move_to_position(mount_location, 270)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[0],\
+            rotated_location[0], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[1],\
+            rotated_location[1], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[2],\
+            rotated_location[2], 6)
+
+    def test360DegreeRotation(self):
+        """
+        Confirm that a SensorMount with a single sensor properly rotates its
+        Sensor object's location when it is rotated 360 degrees without a
+        translation.
+        """
+        mount_under_test = SensorMount()
+
+        sensor_location = (1, 0, 0)
+        attached_sensor = LightSensor(sensor_location, (), ())
+
+        mount_location = (0, 0, 0)
+        rotated_location = (1, 0, 0)
+        mount_under_test.add_new_sensor(attached_sensor)
+        mount_under_test.move_to_position(mount_location, 360)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[0],\
+            rotated_location[0], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[1],\
+            rotated_location[1], 6)
+        self.assertAlmostEqual(mount_under_test.get_sensor(0).current_position()[2],\
+            rotated_location[2], 6)
+
+    def testTranslationAndRotationWithSensor(self):
+        """
+        Confirm that a SensorMount with a single sensor properly updates its
+        Sensor's location when the SensorMount is both rotated and translated.
+        """
+        mount_under_test = SensorMount()
+
+        sensor_location = (100, 0, 0)
+        attached_sensor = LightSensor(sensor_location, (), ())
+
+        mount_location = (2, 5, 7)
+        rotated_location = (86.804809616, -47.991926423, 7)
```

Comment:
That is actually the expected location after the rotation and translation. The comment in the more recent commit clarifies how we independently can confirm those numbers, but in addition I'm going to change the name of this variable from rotated_location to final_location so that we know it isn't just a rotation involved.

Reply:
See if the change in commit 5c48e2b93dcae helps this